PostFreeSurfer/scripts/CreateMyelinMaps.sh

```diff
@@ -92,7 +92,6 @@ Jacobian="${36}"
 ReferenceMyelinMaps="${37}"
 CorrectionSigma="${38}"
 RegName="${39}"
-UseIndMean="${40}"
```

Comment:
This is a problem, as it removes a feature we added.  If your edits to a particular file are small, another thing you can try is to locate the approximate commit that you started editing from, make a branch starting at that commit, commit your edits to that branch, and then use rebase or cherry-pick to make git try to make the same edits starting from Washington-University/NHP_merge.